src/pudl_archiver/frictionless.py

```diff
@@ -160,3 +160,32 @@ def new_datapackage(
             description=data_source.description,
             version=version,
         )
+
+    @classmethod
```

Comment:
This is a hacky method to work around relying on metadata from PUDL. It's pretty ugly/bad, but I think it's ok for the time being until we come up with a better solution for our metadata handling.

Reply:
That time might be a long time in the future... this is to avoid throwing this metadata into `pudl.metadata.resources` basically?